governance.md

```diff
@@ -1,36 +1,50 @@
+* Hold product and code quality in the highest regard.
+* Exhibit stellar judgement and communication.
+* Participate with the project on a consistent basis.
 
-[1]: http://github.com/wp-cli/wp-cli.github.com
-[2]: http://github.com/wp-cli/package-index
-[3]: http://github.com/wp-cli/builds
+Becoming a committer can seem glamorous, but it also comes with expectations of responsibility, commitment, and humility.
 
-If you [subscribe for email updates](https://make.wordpress.org/cli/subscribe/), your email address is registered with WordPress.com through Automattic's Jetpack plugin.
+[andreascreten](https://github.com/andreascreten) is the original author of WP-CLI, and [scribu](https://github.com/scribu) was a long time maintainer.
+
+## What's the connection between WP-CLI and WordPress?
+
+As of December 2017, WP-CLI is a [formal WordPress project](https://make.wordpress.org/core/2016/12/28/supporting-the-future-of-wp-cli/).
+
+Practically-speaking, this means we get to take advantage of the WordPress project's people and software infrastructure:
+
+* Much of the WP-CLI documentation is hosted on the wordpress.org domain.
+* We get to use the `#cli` channel in the [WordPress Slack organization](https://make.wordpress.org/chat/).
+* If we need help with something, we can ask.
+* A couple core committers ([dd32](https://github.com/dd32) and [pento](https://github.com/pento)) have ownership permissions to the WP-CLI GitHub organization, in the unlikely event everyone else gets hit by a bus.
+
+Other than this, WP-CLI operates independently.
 
-### Why are some contributors [members](https://github.com/orgs/wp-cli/people) of the wp-cli organization?
+## How does all of this work?
 
-Because they have demonstrated, over a period of time, that they are capable and invested in moving the project forward:
+The [WP-CLI GitHub organization](https://github.com/wp-cli) contains all project code repositories. [wp-cli/wp-cli](https://github.com/wp-cli/wp-cli) is the main project repository, which pulls in command packages and other dependencies through Composer.
 
-* contributing to the wp-cli codebase
-* promoting wp-cli (speaking at WordCamps, writing blog posts etc.)
-* helping other users and/or contributors
+Some commands are bundled with WP-CLI, while others are independently installable. Commands are bundled with WP-CLI when they're stable, closely aligned with the project philosophy, and useful to a majority of users.
+
+Nightly Phar builds are created by a Travis job that pushes to the [builds repository](http://github.com/wp-cli/builds). Releases are prepared manually [in accordance to the release checklist](https://make.wordpress.org/cli/handbook/release-checklist/).
+
+The `wp-cli.org` domain currently owned by [andreascreten](https://github.com/andreascreten). DNS is managed through a [Cloudflare](https://www.cloudflare.com/) account that [danielbachhuber](https://github.com/danielbachhuber) holds credentials to.
```

Comment:
Fixed in ea67e103ccdd5e29dfb79db25926606f21dab65a

Btw, for small changes like this and https://github.com/wp-cli/core-command/pull/7/commits/32c1989455ff10893419d7ca8616b242c130e8b0, it's more effective to have you make the change (with GitHub's web editor) than go back and forth.